sharpy/structure/utils/lagrangeconstraints.py

```diff
@@ -117,17 +188,12 @@ def define_node_dof(MB_beam, node_body, num_node):
     Define the position of the first degree of freedom associated to a certain node
 
     Args:
-        MB_beam(list): list of 'Beam'
+        MB_beam(list): list of ``Beam``
```

Comment:
If `Beam` refers to a SHARPy class, you can link to it using the syntax `:class:'sharpy.structure.models.beam.Beam'` (note that the apostrophe that should be used is as in the code example below (thanks Markdown formatting)). If you put `~` at the start (as in the example), only `Beam` is shown rather than the full path.
```suggestion
        MB_beam(list): list of :class:`~sharpy.structure.models.beam.Beam`
```